src/Controller/TagController.php

```diff
@@ -4,16 +4,16 @@
 
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
-use Symfony\Component\Routing\Annotation\Route;
 
 class TagController extends AbstractController
 {
 
     public function index(): Response
     {
+        $repository = $this->getDoctrine()->getRepository('App:Tag');
+
         return $this->json([
-            'message' => 'Welcome to your new controller!',
-            'path' => 'src/Controller/TagController.php',
+            'data'=>$tags = $repository->getAll()
```

Comment:
```suggestion
            'data' => $repository->findAll()
```
findAll() - это стандартный метод, делает все тоже самое, как и твой кастомный getAll